operator/pkg/secretsync/cell.go

```diff
@@ -40,6 +48,15 @@ type secretSyncParams struct {
 
 	CtrlRuntimeManager ctrlRuntime.Manager
 	Registrations      []*SecretSyncRegistration `group:"secretSyncRegistrations"`
+	SecretSyncConfig   secretSyncConfig
+}
+
+type secretSyncConfig struct {
+	SecretSyncResyncInterval time.Duration `mapstructure:"secret-sync-resync-interval"`
```

Comment:
no need for `mapstructure` struct tag if the name of the struct field already matches (by convention).